corehq/apps/translations/generators.py

```diff
@@ -230,7 +233,9 @@ def occurrence(_row):
             if type_and_id.type == "Module":
                 case_property_index = self._get_header_index(sheet_name, 'case_property')
                 list_or_detail_index = self._get_header_index(sheet_name, 'list_or_detail')
-                rows = self._filter_invalid_rows_for_module(rows, type_and_id.id, case_property_index, list_or_detail_index)
+                default_en_index = self._get_header_index(sheet_name, 'default_en')
```

Comment:
en would not be the default language always so should do https://github.com/dimagi/commcare-hq/blob/9228f6a6e0e707c1c28cef4f2d98885ef860c001/corehq/apps/translations/generators.py#L197